public/pages/Alerts/containers/Alerts/Alerts.tsx

```diff
@@ -76,15 +81,21 @@ export interface AlertsState {
   groupBy: string;
   recentlyUsedRanges: DurationRange[];
   selectedItems: AlertItem[];
+  correlatedItems: CorrelationAlertTableItem[];
   alerts: AlertItem[];
+  correlationAlerts: CorrelationAlertTableItem[];
   flyoutData?: { alertItem: AlertItem };
+  flyoutCorrelationData?: { alertItem: CorrelationAlertTableItem };
   alertsFiltered: boolean;
+  filteredCorrelationAlerts: CorrelationAlertTableItem[];
   filteredAlerts: AlertItem[];
   detectors: { [key: string]: Detector };
   loading: boolean;
   timeUnit: TimeUnit;
   dateFormat: string;
   widgetEmptyMessage: React.ReactNode | undefined;
+  widgetEmptyCorrelationMessage: React.ReactNode | undefined;
+  tab: string;
```

Comment:
Make this more specific? That way we can have a switch case instead of if... else...
`'detector findings' |  'correlations'`

Reply:
addressed